mkdocs.yml

```diff
@@ -0,0 +1,85 @@
+site_name: Autoware Documentation
```

Comment:
What's this file? I think you should remove it.

Reply:
Removed: https://github.com/autowarefoundation/autoware-documentation/pull/203/commits/381554e6a55cf54352d4374493015d9be283a8d6
(I forgot why I added this file)